extensions/notebook/src/test/book/remoteBook.test.ts

```diff
@@ -0,0 +1,99 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { RemoteBookDialogModel, RemoteBookDialog } from '../../dialog/remoteBookDialog';
+import { RemoteBookController } from '../../book/remoteBookController';
+import * as should from 'should';
+import * as request from 'request';
+import * as sinon from 'sinon';
+
+export interface IExpectedBookItem {
+	title: string;
+	url?: string;
+	sections?: any[];
+	external?: boolean;
+	previousUri?: string | undefined;
+	nextUri?: string | undefined;
+}
+
+describe('Add Remote Book Dialog', function () {
+	let remoteBookDialogModel = new RemoteBookDialogModel();
+	let remoteBookController = new RemoteBookController(remoteBookDialogModel);
+	let remoteBookDialog = new RemoteBookDialog(remoteBookController);
+
+	afterEach(function (): void {
+		sinon.restore();
+	});
+
+	it('Should open dialog successfully ', async function (): Promise<void> {
+		let spy = sinon.spy();
+		spy(remoteBookDialog, 'createDialog');
```

Comment:
You should just be able to do `const spy = sinon.spy(remoteBookDialog, 'createDialog');`